titration/utils/devices/ph_probe.py

```diff
@@ -23,3 +23,8 @@ def set_gain(self, gain):
 
     def get_gain(self):
         return self.ads.gain
+
+    def read_raw_pH(self):
```

Comment:
Is this just an alternate name for `voltage()`. Why not just change the name?

Reply:
I see ... interfaces added this function. Let's address this in a future update. Please add an issue to remove/rename this function?

Reply:
I'll add an issue.